src/cxx_supportlib/WebSocketCommandReverseServer.h

```diff
@@ -256,8 +255,8 @@ class WebSocketCommandReverseServer {
 	};
 
 	struct ConfigChangeRequest {
-		boost::scoped_ptr<ConfigKit::Store> config;
-		boost::scoped_ptr<ConfigRealization> configRlz;
+		boost::asio::detail::scoped_ptr<ConfigKit::Store> config;
```

Comment:
We should use std::unique_ptr instead of anything in a `detail` namespace (because `detail` means private).

But no matter: it's even better to just get rid of WebSocketCommandReverseServer.h because it's only used by AdminPanelConnector.h.